API.md

```diff
@@ -259,6 +259,10 @@ in the [API.md](https://github.com/containers/libpod/blob/master/API.md) file in
 
 [error ContainerNotFound](#ContainerNotFound)
 
+[error ErrCtrStateInvalid](#ErrCtrStateInvalid)
```

Comment:
Wait I added this error already

Reply:
error InvalidState (id: string, reason: string)

Reply:
not for varlink